src/lib/data/constant.ts

```diff
@@ -45,11 +44,11 @@ export const getMaxCodeNameLengthError = (currentLength: number) =>
 export const getListIcon = (listName: string) => {
```

Comment:
add return type `keyof typeof ICONS` and remove it from all return statements

Reply:
Defining a new type called `IconKeys` with the value of `keyof typeof ICONS` in `CustomIcon.tsx` and then utilizing it in the `getListIcon` function as follows:
```tsx
export const getListIcon = (listName: string): IconKeys => {
  switch (listName) {
    case INSTANTIATED_LIST_NAME:
      return "wallet";
    case SAVED_LIST_NAME:
      return "bookmark-solid";
    default:
      return "contract-list";
  }
};
```
is a recommended practice. By using `IconKeys` type as the return value of `getListIcon`, it ensures that the returned value is a valid key from the `ICONS` object. This approach helps to prevent potential errors caused by typos or incorrect key names being returned from the `getListIcon` function.